pennylane/resource/second_quantization.py

```diff
@@ -95,8 +80,7 @@ class DoubleFactorization(Operation):
 
             mol = qml.qchem.Molecule(symbols, geometry, basis_name='sto-3g')
 
-            core_, one, two_ph = qml.qchem.electron_integrals(mol)()
-            two = np.swapaxes(two_ph, 1, 3) # convert to chemist notation
+            core_, one, two = qml.qchem.electron_integrals(mol)()
```

Comment:
Since this is the usage details for the `DoubleFactorization` class, I thought it is more appropriate to create the class instance and access the information from that. I believe majority of users will follow this way which is much more straightforward than calling the methods individually. However if they want to access a method individually, then they can just directly call it without initiating the class.